src/Api.php

```diff
@@ -475,9 +475,10 @@ public static function getWidgetScriptUrl()
 
     /**
      * @param bool $frontendHost
+     * @param string|null $apiHost
      * @return string
      */
-    private static function getApiHost($frontendHost = false)
+    public static function getApiHost($frontendHost = false, $apiHost = null)
```

Comment:
Można sterować łatwo envem zamiast parametry?

Reply:
To nie takie proste. Tutaj musiałem to dodać, aby wyeliminować błąd wołania niewłaściwego hosta API w czasie zapisu zmian w konfiguracji, na podstawie tego co leci w aktualnym formularzu konfiguracji, a nie tego co jest aktualnie w bazie. Stąd to całe zamieszanie tam.